### ISSUE TITLE
PTN000429471 pyruvate carboxylase - subcellular location

### ISSUE THREAD

**Issue by rozaru** [geneontology/go-annotation#5466]
* **PTHR ID & PTN node:**
PTN000429471
* **Sequences with problematic annotation (ID + gene/protein name):**
Q0E9E2 Pcb Pyruvate carboxylase

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**
For GO term cytoplasm there is a direct annotation for drosophila, mouse, human and C.elegans based on yeast data.
While for yeast (and probably other fungi) the protein is localised in the cytosol, for the species mentioned above the protein is actually in the mitochondrion (see human [P11498](https://www.uniprot.org/uniprotkb/P11498/entry)). I couldn't find evidence for a potential cytosolic localisation.

**Comment by pgaudet** [comment]
@rozaru 

Thanks for reporting this - just wanted to point out that this annotation is not wrong, as 'cytoplasm' encompasses both cytosol and mitochondrion, since the mitochondrion is part of the cytoplasm.

**Comment by rozaru** [comment]
You're right @pgaudet. The annotation to cytoplasm is often a pain. Based on the paper evidence curators tend to annotate the protein to the cytoplasm when actually it is likely in the cytosol.
I had a closer look for this protein and according to PMID: 3116940 the yeast proteins PYC1/PYC2 are in the cytosol. Sequence analysis suggests that they don't have a mitochondria signal peptide (at least not a conventional one).
Is it the case for S. Pombe too @ValWood?
If I update the annotation of yeast entries would it help with the PAINT annotation?

**Comment by rozaru** [comment]
Thanks for the explanation on how the cytoplasm annotation came to be. It made sense.

> I can propose to annotate vertebrates with mitochondrion and fungi with cytosol. Would this be okay for you ?

I would agree with this more specific annotation. Could the mitochondrion annotation cover Drosophila and C.elegans too?

**Comment by ValWood** [comment]
SGD reports cytosol for both copies.

The pombe data is HTP to cytosol, but that is all we have right now:
https://www2.riken.jp/SPD/Img_page/41_iP/41A07_Loc.html

I have asked an expert to see if this can be confirmed.

**Comment by rozaru** [comment]
It's an old paper but in PMID: 3116940 they looked at the localisation because there was a controversy about the localisation. It's just that this enzyme is part of gluconeogenesis and I thought it was important if fungi start the pathway in a different compartement.

**Comment by ValWood** [comment]
My carbon  expert says  in pombe nobody has shown properly where it is!

**Comment by deustp01** [comment]
> this enzyme is part of gluconeogenesis and I thought it was important if fungi start the pathway in a different compartment.

Gluconeogenesis  in mammals is mostly cytosolic. It builds glucose molecules from carbon skeletons derived mostly from lactate and amino acid catabolism, and getting those carbon skeletons into the right form to feed gluconeogenesis involves mitochondrial steps and shuttles (e.g., [R-HSA-70263](https://reactome.org/content/detail/R-HSA-70263) with apologies for self-promotion). An important caveat is that different taxa use the process in quite different ways, with quite different dependencies on kinds of carbon skeletons to start it - a fasting adult human uses it heavily while a human embryo depends on the mother for glucose supply, and a chicken embryo developing in an egg must synthesize it from materials in the egg. But as far as I know there's always mitochondrial involvement at the start of the process, cytosolic involvement in the middle, and then ER invoilvement in getting the glucose-6-phosphate generated in the process converted to glucose and exported from the cells doing gluconeogenesis to make it available to the rest of the body. [end of biochem lecture]

**Comment by marcfeuermann** [comment]
In my opinion, it is indeed a classical mitochondrial function in eukaryotes. However, as the annotations for fungi were all to cytosol, (4 different annotations), I wondered whether we might be dealing with a family containing cytosolic isoforms for fungi. I am tempted to propagate the mitochondrial location to all eukaryotes... if you agree with this.

**Comment by ValWood** [comment]
Maybe ask somebody from SGD to check if they have any data for this. @edwong57  could you assign to the appropriate person. If it is mitochondrial in yeast, I assume there is some paper somewhere to support this?

**Comment by deustp01** [comment]
> If it is mitochondrial in yeast, I assume there is some paper somewhere to support this?

According to Reed and Hackert (1990 - PMID: 2188967; the Voice of God on this subject), "In eukaryotic cells, the alpha-keto acid dehydrogenase complexes are located in mitochondria within the inner membrane-matrix compartment." This review mentions yeast, probably S. cerevisiae, at several points so that might lead to experimental data.

**Comment by marcfeuermann** [comment]
I will check this and look for more evidence when I will have some more time. For sure there is. I will probably extend the mitochondrial annotation then. I let you know before closing this ticket.
